### PR TITLE
Accept multiple files.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,3 +32,26 @@ jobs:
       - name: Check signature
         run: |
           codesign -dvvv main
+
+      - name: Build executable
```
Comment (kasperl) [inline .github/workflows/ci.yml]: Maybe distinguish the names of the actions?

**Review by kasperl — APPROVED** [review]
Looks good enough.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (floitsch) [reply .github/workflows/ci.yml]: done.

**Review by kasperl — APPROVED** [review]
Looks good to me.

**Pull request merged**